connectors/opa/lib/opa_reader.go

```diff
@@ -37,7 +44,16 @@ func (r *OpaReader) GetOPADecisions(in *pb.ApplicationContext, catalogReader *Ca
 	if err != nil {
 		return nil, fmt.Errorf("error in unmarshalling appInfoBytes: %v", err)
 	}
-
+	// get other properties from vault
+	var jsonStr string
+	if jsonStr, err = r.vaultConnection.GetSecret(os.Getenv("VAULT_USER_HOME") + in.AppId + "/" + os.Getenv("CATALOG_PROVIDER_NAME")); err != nil {
+		return nil, err
+	}
+	creds := make(map[string]string)
+	if err = json.Unmarshal([]byte(jsonStr), &creds); err != nil {
+		return nil, err
+	}
```

Comment:
I don't remember why it was decided to pass appId instead of the full path. On the other hand, it would be preferable for the connector to read the secret directly.

Reply:
No longer used by the connector.